src/app/util/ContentApp.cpp

```diff
@@ -46,20 +46,43 @@ namespace AppPlatform {
 EmberAfStatus ContentApp::HandleReadAttribute(ClusterId clusterId, chip::AttributeId attributeId, uint8_t * buffer,
                                               uint16_t maxReadLength)
 {
-    ChipLogProgress(DeviceLayer, "Read Attribute for device %s cluster %d attribute=%d)",
+    ChipLogProgress(DeviceLayer, "Read Attribute for device %s cluster %d attribute=%d",
                     GetApplicationBasic()->GetApplicationName(), static_cast<uint16_t>(clusterId),
                     static_cast<uint16_t>(attributeId));
 
-    EmberAfStatus ret = EMBER_ZCL_STATUS_FAILURE;
     if (clusterId == ZCL_APPLICATION_BASIC_CLUSTER_ID)
     {
-        ret = GetApplicationBasic()->HandleReadAttribute(attributeId, buffer, maxReadLength);
+        return GetApplicationBasic()->HandleReadAttribute(attributeId, buffer, maxReadLength);
+    }
+    if (clusterId == ZCL_ACCOUNT_LOGIN_CLUSTER_ID)
+    {
+        return GetAccountLogin()->HandleReadAttribute(attributeId, buffer, maxReadLength);
+    }
+    if (clusterId == ZCL_ACCOUNT_LOGIN_CLUSTER_ID)
```

Comment:
Why are we testing the same thing over and over?

Reply:
sorry, you caught this work-in-progress. Just fixed this but I am planning to refactor this code in this PR in order to migrate to AttributeAccessInterface